client/app/scripts/superdesk-search/views/search-results.html

```diff
@@ -22,8 +22,6 @@
         <div class="content" sd-updown data-items="items._items" data-select="preview(item)">
             <div class="loading-indicator" ng-show="loading" translate>loading</div>
 
-            <div sd-search-tags></div>
```

Comment:
This little forgotten thing here caused all the problems. :wink: It was this `sd-search-tags` directive triggering the missing controller error, not the one we were looking at.